src/BayesFilters/include/BayesFilters/SimulatedStateModel.h

```diff
@@ -17,7 +17,7 @@ class bfl::SimulatedStateModel : public Agent, public Logger
 {
 public:
     SimulatedStateModel(std::unique_ptr<StateModel> state_model,
```

Comment:
Let's have all the signatures in the header on one line for now 👍
(Yes, probably this was already multi-line. My bad! 😬)